waka_waka/Assets/maze_spritemap.tres

```diff
@@ -0,0 +1,46 @@
+[gd_resource type="TileSet" load_steps=4 format=3 uid="uid://2kbf0ykmvv3r"]
+
+[ext_resource type="Texture2D" uid="uid://c2kq5cj4oe0it" path="res://Assets/maze_spritemap.png" id="1_4sko1"]
+
+[sub_resource type="PhysicsMaterial" id="PhysicsMaterial_kupj4"]
+friction = 0.6
+bounce = 0.19
+absorbent = true
+
```

Comment:
This is very ugly. Not anything needs to be done, but tilemaps should probably definitely be in the godot editor and not manually.